dist/enhanced-shutter-card.js

```diff
@@ -1219,12 +1245,12 @@ class EnhancedShutter extends LitElement
     this.pickPoint = event.pageY - parseInt(slide.style.height);
   }
   getShutterPosition(newScreenPosition){
-    let shutterPosition = (newScreenPosition - this.cfg.topOffsetPct()) * (100-this.cfg.offset()) / this.cfg.coverHeightPx();
+    let shutterPosition = (newScreenPosition ) * (100-this.cfg.offset()) / (this.cfg.coverSlatsAreaHeightPx());
```

Comment:
Explain this change

Reply:
The behaviour changes due to the margins on the shutter area.

Reply:
I pushed a fix. I missed the shutterBottomHeight. Fixed the initial jump when dragging.